scripts/histogramming_config.py

```diff
@@ -0,0 +1,60 @@
+import Histogramming.storage_functions as sf
```

Comment:
This should be in a directory called configs which is parallel to scripts and utils

Reply:
moved to configs